apps/emqx_management/i18n/emqx_mgmt_api_alarms_i18n.conf

```diff
@@ -62,7 +62,7 @@ The default is false."""
 
     duration {
         desc {
-            en: """Indicates how long the alarm has lasted, in milliseconds."""
+            en: """Indicates how long the alarm has been active in milliseconds."""
```

Comment:
Just a question: why do we still mention _milliseconds_ here?

Reply:
this value is a `integeter()` type, but not `emqx_schema:duration_ms()` or similar.